### PR TITLE
Forbidden duplicated method parameters

### PR DESCRIPTION
**Pull request by shuheiktgw**
I just noticed goby allows to define methods with the same argument name, which is prohibited in ruby. An example below

```
def someMethod(a, a)
  return a
end
```

So I just disabled it! Thanks! 😄

### CODE REVIEW

**Comment by shuheiktgw** [comment]
I just finished implementing it, so could anyone review this PR?

File: compiler/parser/statement_parsing.go [hunk 1]
```diff
@@ -117,6 +117,10 @@ func (p *Parser) parseParameters() []ast.Expression {
 		p.nextToken()
 		p.nextToken()
 		param := p.parseExpression(NORMAL)
+
+		if containsArg(params, param) {
```
Comment (st0012) [inline compiler/parser/statement_parsing.go]: I recommend renaming it to `paramDuplicated` which will be more clear as a condition.

**Comment by st0012** [comment]
I didn't notice this, thanks! And remember to fix codeclimate's warning.

**Comment by shuheiktgw** [comment]
Oops, sorry I just fixed the problems mentioned above!
Thanks for pointing out! 👍

**Pull request merged**